src/plugins/data/server/search/strategies/ese_search/request_utils.ts

```diff
@@ -46,21 +46,18 @@ export async function getDefaultAsyncSubmitParams(
     | 'keep_on_completion'
   >
 > {
+  // TODO: searchSessionsConfig could be "null" if we are running without x-pack which happens only in tests.
+  // This can be cleaned up when we completely stop separating basic and oss
+  const keepAlive = searchSessionsConfig?.enabled
+    ? `${searchSessionsConfig.defaultExpiration.asMilliseconds()}ms`
+    : '1m';
   return {
     batched_reduce_size: 64,
     keep_on_completion: !!options.sessionId, // Always return an ID, even if the request completes quickly
```

Comment:
Should we also make this conditional? 
Something like: 
```suggestion
    keep_on_completion: searchSessionsConfig?.enabled ? !!options.sessionId : false,
```

Reply:
:+1: 9766368d765a98945e5db29f83ab66f9177d4be2